rb/spec/integration/selenium/webdriver/firefox/profile_spec.rb

```diff
@@ -22,137 +22,161 @@
 module Selenium
   module WebDriver
     module Firefox
-      compliant_on driver: :ff_legacy do
```

Comment:
Are these specs specified to be run outside firefox tests? I was under the impression that it was only included in `firefox-test`, `ff-legacy-test`, `remote-firefox-test`, and `remote-ff-legacy-test`.

The tests should work for remote as well